### ISSUE TITLE
Build Fails When Running Nine00SDKTesterSource

### ISSUE THREAD

**Issue by CramLabs** [900Seconds/900SecondsSDK#4]
So I downloaded the project and ran `pod setup` and `pod install` on the project folder (Nine00SecondsSDKExample) When I try to run it I get these three errors. 
- `clang: error: no such file or directory:  '/Users/myname/Desktop/Submodules/libextobjc/extobjc/EXTScope.m'`
- `clang: error: no such file or directory: '/Users/myname/Desktop/Submodules/libextobjc/extobjc/EXTSelectorChecking.m'`
-  `No input files`

In conclusion what I am trying to ask is there something I'm missing here in installation I am not able to test out the demo project. Or do you have any suggestions on how I can fix this?

**Comment by aziz92** [comment]
Getting the same issue. Seems like this project is dead.

**Comment by dangerden** [comment]
@aziz92 @CramLabs sorry for the delay, will be updated this week.

**Comment by IPv6** [comment]
@aziz92 @CramLabs fixed, please checkout/download latest version
we simplified external dependencies, now test project contain everything to build "right away"
